conductor/submitter.py

```diff
@@ -1936,6 +1988,16 @@ def get_next_frames_chunk(self, chunk_size, uniform_chunk_step=True):
 
         return task_frames
 
+
```

Comment:
Yes, a `count` makes sense and describes that it doesn't know the length before counting. I'll do it another way. 
Should your example be:
`self._task_count += 1 if next_task_frames else 0` 
since next_task_frames is a tuple with 4 elements or None?

Reply:
ah yes, good catch!

In terms of the `count` attribute name, I was wavering between `count` and `task_count`.  But because this class equally addresses both _tasks_ and _frames_, I thought it might be unclear as to what _count_ refers to.  Hence `task_count`.  This would also open up a logical slot for _frame_count_  in the future.  What do you think?

Reply:
On another note - I'm coming to the conclusion that the comments in this class describe different behavior. Looks like it doesn't dispense commands anymore, just the numbers needed to make them.

Reply:
ha, I noticed that too. I was very tempted to change them just now (as well and move the entire class to another module), but didn't want to create merge conflicts :)

Reply:
Yes I can make it task_count. When I look at this class, my first thought is that it deals with tasks, which is why I thought `len` was a good fit for returning the number of tasks - then again, it has `Frames` in the name, so a little ambiguous. Really it's a chunk generator, now that it no longer knows about commands. `chunks.count()` seems natural -  and for frame_count: `sum([len(c) for c in chunks])` where c is a `range` object. Because I notice the 4th element in the tuple is the frames themselves, so it's kind of redundant given that we have `start/end/step` already. Having said all that, I can imagine there are reasons and stuff I'm not aware of :)

Reply:
How about, for now, I'll remove __len__ and just instantiate a TaskFrameGenerator for the purpose of counting it? 
```
task_counter =  TaskFramesGenerator( frames, chunk_size=chunk_size, uniform_chunk_step=True)
task_count = sum([1 for _ in task_counter])
```